src/rfcml_bending/bending.py

```diff
@@ -165,12 +165,20 @@ def bending_mode_correction(
         forces: array
             Array of forces on each actuator
 
+        bendCoef: array
+            array of coefficients of linear combination of bending modes
+
         """
 
         assert mask.dtype == bool, "Mask needs to be a dtype of boolean."
 
         # set terms in decomposed Zernike surface to zero to reconstruct a map without piston/tip/tilt/focus/coma (default)
-        removed_map, zern_fit_map = remove_zerns(map, mask, rm_zerns)
+        if gauss_remove == True:
+            removed_map, zern_fit_map = remove_zerns(map, mask, rm_zerns)
+        else:
+            print('No Zern removal')
```

Comment:
Doh... i was going to grumble that you didn't use the logger, except I see there isn't one setup for this package. 
Maybe add a more concise message
```suggestion
            print('Not removing zernikes because gauss_remove=False')
```